### PR TITLE
Make private dupe message more descriptive

### PR DESCRIPTION
**Pull request by NeunEinser**

### CODE REVIEW

**Comment by Pokechu22** [comment]
There should probably be a period after "access it".

**Comment by NeunEinser** [comment]
> 
> 
> The name is currently "Duplicate (private)". Maybe it should be more clear that this is for exploits and therefore the name should be "Duplicate (private exploit)" instead?
> 
> However, it also depends on when this message is used. Maybe staff members use it also if the duplicated report is private for other reasons, e.g. because the reporter does not want their world to become public.
> In that case, this pull request should add a separate message.

From the mod workflow, we are supposed to prefer public tickets as parent, so I don't see this message being used in any other circumstance.
Do you have an example where the old message was actually used in the described context?

**Comment by RickTheGeezer** [comment]
This is Auldrick. I asked Marcono to comment because I'm not a member on this team and have never collaborated via GitHub, so I wasn't sure I would have time to learn how to leave a comment.

1. Use of Private on an exploit does not preclude its use for other purposes. It is not _reserved_ for exploits.
1. MC and MCPE have different mod workflows. In MCPE we use Private to avoid unnecessary exposure of gamertags and other personal information, especially in screenshots, and to block access to attached worlds the reporter might not wish to share, such as Marketplace worlds in development.

Avoiding changing the helper message hardly seems like a justification for losing the flexibility inherent in the Private security attribute.

**Comment by NeunEinser** [comment]
I can add another message for MCPE.
I only work on MC, so I wouldn't be aware of such differences.

**Comment by Marcono1234** [comment]
>From the mod workflow, we are supposed to prefer public tickets as parent, so I don't see this message being used in any other circumstance.

I guess there could be cases where we prefer to keep the private report open, e.g. because it is more detailed or has the required data for reproducing the bug (e.g. attached world) or a developer already assigned themselves to the private report.

In case you create two separate messages it would probably not hurt to allow both to be used by all projects and not restrict one to MCPE.

File: assets/messages.yml [hunk 1]
```diff
@@ -316,7 +316,9 @@ messages:
       name: Duplicate (private)
       message: |-
         *Thank you for your report!*
-        We're actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
+        We''re actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
```
Comment (Marcono1234) [inline assets/messages.yml:319]:
```suggestion
        We're actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
```

Same as https://github.com/mojira/helper-messages/pull/42#discussion_r426697671

**Pull request merged**